src/pages/BatchSearch.vue

```diff
@@ -34,23 +34,23 @@ export default {
   },
   mixins: [utils],
   computed: {
-    ...mapState('batchSearch', ['hasBatchSearch']),
+    ...mapGetters('batchSearch', ['hasBatchSearch']),
     howToLink() {
       return '#/docs/all-batch-search-documents'
     },
     noBatchSearch() {
       return this.$t('batchSearch.empty', { howToLink: this.howToLink })
     }
   },
-  async mounted() {
+  mounted() {
     if (!this.hasBatchSearch) {
-      await this.getBatchSearch()
+      this.getBatchSearches()
```

Comment:
`getBatchSearches()` is async right ?

Reply:
In this case the promise does not need to be "awaited" since it's in the `mounted` hook and there is nothing to do after that requires to await the result of the promise.